### PR TITLE
center aligning text in ticket section for mobile view

### LINKED ISSUE #358: Center Align ticket section text for mobile view
image

### PR DESCRIPTION
**Pull request by 142ayushkumar**
Closes #358 
Preview: 
image
image

### CODE REVIEW

**Comment by 142ayushkumar** [comment]
Deployed [here](https://142ayushkumar.github.io/deploy/)

**Comment by gabru-md** [comment]
ping @mariobehling 
merge please?

File: css/custom.css [hunk 1]
```diff
@@ -177,3 +177,8 @@
     clear: left;
   }
 }
+
+.text-middle {
```
Comment (realslimshanky) [inline css/custom.css]: This is not required. Bootstrap already has a class `text-center` which you can use straight away.

File: index.html [hunk 2]
```diff
@@ -182,7 +182,7 @@ <h1 class="large-h1 text-white"><span>"Open Technologies for Personal Voice Assi
                             </div>
 
                             <div class="col-md-4 col-sm-8">
-                                <h1>Get your Ticket for the event now here!</h1>
+                                <h1 class="text-middle">Get your Ticket for the event now here!</h1>
```
Comment (realslimshanky) [inline index.html]: Use `text-center` instead.

File: index.html [hunk 3]
```diff
@@ -192,7 +192,7 @@ <h1>Get your Ticket for the event now here!</h1>
                                 <i class="icon pe-7s-id icon-large"></i>
                             </div>
 
-                           <div class="col-md-4 col-sm-8">
+                           <div class="col-md-4 col-sm-8 text-middle">
```
Comment (realslimshanky) [inline index.html]: Use `text-center` instead on child `h1`.

File: index.html [hunk 4]
```diff
@@ -192,7 +192,7 @@ <h1>Get your Ticket for the event now here!</h1>
...
                                 <h1>Check out the schedule with 200+ sessions from speakers around the world from the previous year.</h1>
```
Comment (realslimshanky) [inline index.html]: Add `class="text-center"` to h1.

Comment (142ayushkumar) [reply index.html]: My bad, actually I have not used bootstrap yet. I will soon update it.
Thanks

**Comment by realslimshanky** [comment]
@142ayushkumar you've added git conflict messages by mistake into the commits. Please remove them.

File: index.html [hunk 5]
```diff
@@ -182,19 +182,17 @@ <h1 class="large-h1 text-white"><span>"Open Technologies for Personal Voice Assi
...
                             </div>
                         </a>
 
                         <a href="https://eventyay.com/e/b548bcfc/" class="block-content" target="_self">
                             <div class="col-md-2 col-sm-4 text-center">
                                 <i class="icon pe-7s-id icon-large"></i>
                             </div>
-
-                           <div class="col-md-4 col-sm-8">
+
```
Comment (realslimshanky) [inline index.html]: Please remove this unnecessary new line.

**Comment by mariobehling** [comment]
Your PR changed the alignment of the text, which should not be changed. Also please see the comment here: https://github.com/fossasia/2018.fossasia.org/issues/358#issuecomment-361748550

Therefore closing it.

**Pull request closed without merging**